README.md

```diff
@@ -41,6 +41,10 @@ To run all tests use:
 cd BladeMonRT
 go test ./...
```

Comment:
See if we can exclude E2E test, if so, let's make it so people don't have to change the filename